tests/virt/utils.py

```
@@ -355,3 +363,112 @@ def validate_machine_type(vm, expected_machine_type):
+                        ]
+                    },
+                }
+            }
+        },
+        list_resource_reconcile=[KubeVirt],
+        wait_for_reconcile_post_update=True,
+    ):
+        yield
+
+
+def fetch_gpu_device_name_from_vm_instance(vm):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)